### PR TITLE
feat: responsive header

### PR DESCRIPTION
**Pull request by IamDikshaR**

### CODE REVIEW

File: public/icons/google_symbol.svg
Comment (tanishdotdev) [inline public/icons/google_symbol.svg:1]: Rename the images in lowercase separated by hyphens.
This should be google-symbol.svg. I think it should be called google-logo.svg.

File: src/atoms-diksha/buttons/solid-button/color-accent/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/buttons/solid-button/color-accent/index.astro:1]: Remove solid-button from the path in all buttons.
It should be src/atoms-diksha/buttons/color-accent/index.astro

File: src/atoms-diksha/headings/five/lg/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/headings/five/lg/index.astro:1]: Add color in the path.
src/atoms-diksha/headings/five/lg/accent/index.astro

File: src/atoms-diksha/headings/six/color-blue/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/headings/six/color-blue/index.astro:1]: Update path: src/atoms-diksha/headings/six/blue/index.astro

File: src/atoms-diksha/headings/six/color-grey/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/headings/six/color-grey/index.astro:1]: Remove color- from all headings.

File: src/atoms-diksha/logo/index.astro
Comment (tanishdotdev) [inline src/atoms-diksha/logo/index.astro:1]: Why are you using direct link of image? download it and add in public.

File: src/components/common/header/index.astro
Comment (tanishdotdev) [inline src/components/common/header/index.astro:1]: 1. Rename the component HeadingFiveLarge.
2. Rename the button as ButtonAccent.
3. Follow the chronology of imports.
LAYOUTS
COMPONENTS
ATOMS
PROPS
4. Format map() correctly.
5. Fix spacing  in curly brackets like { item } .
6. Use link atom inplace of anchor.

File: src/components/common/header/nav-items/dropdown-items/index.astro
Comment (tanishdotdev) [inline src/components/common/header/nav-items/dropdown-items/index.astro:1]: 1. Use link atom inplace of anchor in all files.
2. Remove semicolons.
3. Fix hoverr.

File: src/components/common/header/nav-items/index.astro
Comment (tanishdotdev) [inline src/components/common/header/nav-items/index.astro:1]: 1. Fix spacing in curly bracket { link }.

File: src/components/home/client-reviews/google-review-card/index.astro
Comment (tanishdotdev) [inline src/components/home/client-reviews/google-review-card/index.astro:1]: 1. Rename HeadingSixBlue, HeadingSixGrey.
2. Use picture atom inplace of image tag.
3. Create button atom.

File: src/components/home/client-reviews/index.astro
Comment (tanishdotdev) [inline src/components/home/client-reviews/index.astro:1]: 1. Follow chronology of imports.
LAYOUTS
COMPONENTS
ATOMS
PROPS

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

File: public/icons/google_symbol.svg
Comment (IamDikshaR) [reply public/icons/google_symbol.svg:1]: done

File: src/atoms-diksha/buttons/solid-button/color-accent/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/buttons/solid-button/color-accent/index.astro:1]: done

File: src/atoms-diksha/headings/five/lg/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/headings/five/lg/index.astro:1]: It's a hover color, doesnt appear on view, should i add that?

File: src/atoms-diksha/headings/six/color-blue/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/headings/six/color-blue/index.astro:1]: done

File: src/atoms-diksha/headings/six/color-grey/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/headings/six/color-grey/index.astro:1]: done

File: src/atoms-diksha/logo/index.astro
Comment (IamDikshaR) [reply src/atoms-diksha/logo/index.astro:1]: done

File: src/components/common/header/index.astro
Comment (IamDikshaR) [reply src/components/common/header/index.astro:1]: done

File: src/components/common/header/nav-items/dropdown-items/index.astro
Comment (IamDikshaR) [reply src/components/common/header/nav-items/dropdown-items/index.astro:1]: done

File: src/components/common/header/nav-items/index.astro
Comment (IamDikshaR) [reply src/components/common/header/nav-items/index.astro:1]: done

File: src/components/home/client-reviews/index.astro
Comment (IamDikshaR) [reply src/components/home/client-reviews/index.astro:1]: done

File: src/components/home/client-reviews/google-review-card/index.astro
Comment (IamDikshaR) [reply src/components/home/client-reviews/google-review-card/index.astro:1]: done

**Pull request closed without merging**